### PR TITLE
Fix integration tests according to changes in protobuf structure

### PR DESCRIPTION
**Pull request by aleksuss**
## Overview

---

See: https://jira.bf.local/browse/ECR-XYZW

### Definition of Done

- [ ] There are no TODOs left in the merged code
- [ ] Change is covered by automated tests
- [ ] Benchmark results are attached (if applicable)
- [ ] The [coding guidelines] are followed
- [ ] Public API has proper documentation
- [ ] Changelog is updated if needed (in case of notable or breaking changes)
- [ ] The continuous integration build passes

[coding guidelines]: https://github.com/exonum/exonum/blob/master/CONTRIBUTING.md#conventions

### CODE REVIEW

File: test-suite/testkit/server/package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   },
   "private": true,
   "scripts": {
-    "proto": "pbjs -t static-module -r test ../../../examples/cryptocurrency/src/proto/service.proto ../../../components/crypto/src/proto/schema/types.proto -o ./test/stubs.js",
+    "proto": "pbjs -t static-module -r test ../../../examples/cryptocurrency/src/proto/service.proto ../../../components/crypto/src/proto/schema/exonum/crypto/types.proto -o ./test/stubs.js",
```
Comment (slowli) [inline test-suite/testkit/server/package.json]: This is completely not your fault, but this command is arguably bogus. It should be changed to something like

```
pbjs -t static-module
  -r test
  -p node_modules/exonum-client/proto
  ../../../examples/cryptocurrency/src/proto/service.proto
  -o test/stubs.js
```

(Line breaks are inserted for readability.) The `-p` option specifies an include path (which, in this case, is a path to files included in the `exonum-client` package).

Comment (aleksuss) [reply test-suite/testkit/server/package.json]: At least it works locally. I'll check the command line locally and change it in the release commit.

Comment (slowli) [reply test-suite/testkit/server/package.json]: Yeah, it works, but the current command is more brittle. The version with an include path will continue working if `service.proto` requires more types from core, or if they are rearranged within the core.

Comment (aleksuss) [reply test-suite/testkit/server/package.json]: But I'll fix it in the upcoming release commit to avoid one more CI flow.

**Pull request merged**